ironfish/src/consensus/consensus.ts

```diff
@@ -83,5 +96,6 @@ export class TestnetParameters extends ConsensusParameters {
   constructor() {
     super()
     this.V1_DOUBLE_SPEND = 204000
+    this.V2_MAX_TRANSACTIONS = 230000
```

Comment:
We could get away from a hard fork declaration because no one has ever made a block with more than 301 transactions which is our miner manager soft cap.